lib/models/collection.dart

```diff
@@ -1,28 +1,25 @@
-// @dart=2.9
-
 import 'dart:convert';
 import 'dart:core';
 
-import 'package:flutter/foundation.dart';
 import 'package:photos/models/magic_metadata.dart';
 
 class Collection {
   final int id;
-  final User owner;
+  final User? owner;
   final String encryptedKey;
-  final String keyDecryptionNonce;
+  final String? keyDecryptionNonce;
   final String name;
```

Comment:
name can be null afaik. At start, we used have name, later on we decided to store encrypted name

Reply:
Have made it nullable